pkg/configurator/types.go

```diff
@@ -84,4 +84,12 @@ type Configurator interface {
 	// GetConfigResyncInterval returns the duration for resync interval.
 	// If error or non-parsable value, returns 0 duration
 	GetConfigResyncInterval() time.Duration
+
+	// POC ExternalAuthz
+	GetOutboundExternalAuthzEnable() bool
+	GetOutboundExternalAuthzAddress() string
+	GetOutboundExternalAuthzPort() int
+	GetOutboundExternalAuthzStatPrefix() string
+	GetOutboundExternalAuthzTimeout() time.Duration
+	GetOutboundExternalAuthzFailureModeAllow() bool
```

Comment:
Individual methods seem unnecessary. Could you return a single struct here?

Reply:
Not final design, thought of this too. Since authorization could be place on different listeners, it's a good idea to have a common struct.